.travis.yml

```diff
@@ -22,7 +23,6 @@ jobs:
       rust: stable
       cache:
         directories:
-          - $HOME/.cache/node-gyp
```

Comment:
So if I'm not mistaken, we started caching `node-gyp` because we would get intermittent failures where it wasn't properly initialized.

I imagine the issues with caching node-gyp are related to changing the NodeJS version?

I'll dig up the PR

Reply:
Here it is: https://github.com/gnosis/dex-services/pull/500

Reply:
Now that I think of it, maybe this was just an issue with NodeJS 10 that is fixed now. I'm all for removing this line for now and if the intermittent failures come back, investigate again.